### PR TITLE
Fix #1883: Add confirmation alerts for resets

### LINKED ISSUE #1883: Show confirmation for buttons that reset progress
As it is now, `Soft reset and Restart tutorial`, `Install Augmentations` and possibly other buttons reset the game, but there is no prompt shown before doing so, so if someone misclicks or misunderstands, their progress is lost.

It'd be nice to add confirmations for those buttons, and a toggle in settings to not show them for experienced players who do not want to get nagged by pop ups.

### PR DESCRIPTION
**Pull request by MartinFournier**
Add a yes/no alert for augmentation installation & soft reset

Augmentations

firefox_YdpAxHJwaJ

Soft Reset

firefox_33G76gBsex

### CODE REVIEW

File: src/Augmentation/ui/AugmentationsRoot.tsx [hunk 1]
```diff
@@ -72,11 +75,34 @@ export function AugmentationsRoot(props: IProps): React.ReactElement {
...
+              <Typography>
+                WARNING: THIS WILL PARTIALLY RESET YOUR PROGRESS
+                <br />
+                <br />
+                For your first installation, it is recommended to purchase most of the CSEC augmentations.
+                <br />
+                If you are unsure, you should EXPORT your save game as a backup before!
+                <br />
+                <br />
+                <Typography variant="subtitle2"><em>This is the first prestige mecanism.</em></Typography>
```
Comment (MageKing17) [inline src/Augmentation/ui/AugmentationsRoot.tsx:92]:
```suggestion
                <Typography variant="subtitle2"><em>This is the first prestige mechanism.</em></Typography>
```
Should this even be mentioned at all?

Comment (MartinFournier) [reply src/Augmentation/ui/AugmentationsRoot.tsx:92]: Possibly not but some player seemed confused by the apparent progress reset. I can remove it if preferred.

Comment (MageKing17) [reply src/Augmentation/ui/AugmentationsRoot.tsx]: I meant the hint that there was more than one prestige mechanism.

**Comment by hydroflame** [comment]
Added but differently, ty though

**Pull request closed without merging**